### ISSUE TITLE
Militia Community and SWAT / Police presence. 

### ISSUE THREAD

**Issue by RanikOrtega** [CleverRaven/Cataclysm-DDA#72832]
### Is your feature request related to a problem? Please describe.

Militia, Preppers and Police are present in game as a starting profession and as feral humans and locations (Prepper houses) however we have no living presence to represent them. Surely at least some of them survived. 

A community of a handful of militia preppers located in a large lodge in the woods. They had plans before the Cataclysm but are now in a much worse state since 95+% of their members went zombie. They consist of current / former military, police and fire rescue members. Initially they are competent and skilled at defending themselves but they are as off guard as everyone else since most members are dead and in need of assistance for practical tasks. Repairs to their location, electric wiring etc etc. Over time you get involved with some of their activities and troubles. 

Police / SWAT presence occurs after several missions and the militia hears calls for help over the radio. Since some of their members are former Police they may know the people on the other side of the radio or at least know of them. 

### Solution you would like.

- [ ] Initial Implementation and Quest 1 - Location of Militia lodge and several NPC's. Militia leader has some construction jobs and perhaps some items for trade. 
- [ ] Quest 2 - Some of the militia members are gone, the leader tells the player they went on a supply run for medical supplies after an attack and now the team is stuck. Player goes to location breadcrumb and frees militia members via hacking open a door or bashing a wall down. 
- [ ] Quest 3 - They report that they were injured trying to access a SWAT team armory that was both more empty of weaponry and more defended by zombies than expected. You are able to find some clue at the location which the member is able to decipher a SWAT team took the supplies. 
- [ ] Quest 4 - Militia Leader later reports that after the armory raid they suspect they know what happened. They were contacted by a SWAT team captain before everything went wrong and declined to join the SWAT team. The militia leader suspects the remainder of a SWAT Team is holed up at a location they had mentioned. The location is the top floor of a unique and potentially fortified skyscraper building. The building was used as the retreat point and last line of defense for the cities police forces before being over-run. The building is now heavily zombie infested and only 1-2 members of the SWAT Team is alive at the top. Once rescued they move to Militia camp. 

### Describe alternatives you have considered.

Additional quests could involve rescuing individual militia members that are stranded. Survivors overheard on radio or raiding locations that they may know of or learn about.

**Comment by I-am-Erk** [comment]
You know, an interesting optional end point for this might be that they offer to hire you permanently as their scout and handyman, with this OOC explicitly stated as "this will end your game", and you give up being a solo adventurer and settle down as a member of their community in a non-leader role.

**Comment by RanikOrtega** [comment]
> You know, an interesting optional end point for this might be that they offer to hire you permanently as their scout and handyman, with this OOC explicitly stated as "this will end your game", and you give up being a solo adventurer and settle down as a member of their community in a non-leader role.

I like the idea. I generally like options that allow you to "End your game" even if I never use them. 

With this one, would that best be locked behind tie in missions to refugee center mission also being completed? IE Refugee center and Militia have made contact and are somewhat working together. 

Or would it be better standalone with the militia questline being finished?

**Comment by kevingranade** [comment]
Flavor notes I mentioned on Discord for Quests 3 and 4:  
Instead of a warehouse, it's a police station with a SWAT team armory, but upon entry, you find that it's already raided of the high-end items they were looking for. You do turn up some cryptic notes that one of the militia members can decipher, which leads to quest 4.  
A SWAT team and their leadership had emptied out the armory and moved to what they thought was a defensible location in a highrise building, but had been overrun. They had contacted one of the militia members about it before SHTF, but they had been unable or unwilling to join them, but have enough context to figure out where the SWAT team had holed up.  
I think two back to back rescue missions is a bit much, it seems more sensible for the penthouse location to just be a looting target, though you may go into it not knowing they had been overrun.  Potentially they would attempt to make contact but then find out that the place is swarming with zombies and evidence that the SWAT team had been overrun and all the gear would be in the penthouse.

**Comment by RanikOrtega** [comment]
> Flavor notes I mentioned on Discord for Quests 3 and 4: Instead of a warehouse, it's a police station with a SWAT team armory, but upon entry, you find that it's already raided of the high-end items they were looking for. You do turn up some cryptic notes that one of the militia members can decipher, which leads to quest 4. A SWAT team and their leadership had emptied out the armory and moved to what they thought was a defensible location in a highrise building, but had been overrun. They had contacted one of the militia members about it before SHTF, but they had been unable or unwilling to join them, but have enough context to figure out where the SWAT team had holed up. I think two back to back rescue missions is a bit much, it seems more sensible for the penthouse location to just be a looting target, though you may go into it not knowing they had been overrun. Potentially they would attempt to make contact but then find out that the place is swarming with zombies and evidence that the SWAT team had been overrun and all the gear would be in the penthouse.

I'll add the flavor notes for the SWAT team armory. 

For the rescue missions. The initial hook was rescuing their people stuck in a building they were raiding for medical supplies. Then the SWAT team holding out on the top floor. 

Personally I like the fetch quest being for medical supplies. Then the SWAT Team rescue raid actually spawning new NPCS in the militia camp after you rescue the 1-2 people. Thoughts?

**Comment by I-am-Erk** [comment]
I agree that growing the camp over time feels cool, better than recruiting new NPCs for your own faction or something. It also doesn't feel as much like a rescue mission since finding them is more of a secondary surprise, you're going there to see if they have ammo

**Comment by PatrikLundell** [comment]
I have some logical reservations on this quest line:
- If they're preppers they ought to have done something to allow for repair/construction/maintenance, both in terms of tools, materials, and instruction manuals. That should help them even if their team is decimated. The way the scenario is described looks more like a military type group stuck together in the middle of nowhere in deep shit because they have military type training, but not much in the way of skills in construction/maintenance/... (which means they're probably all city people, as rural people tend to have general practical skills) The logistics people weren't part of their team...
- Why would SWAT forces use a random nobody for a combat rescue mission? THEY are the ones who have combat training (and equipment), not the random survivor. It would make much more sense to have their group secure a location they know or suspect have useful stuff and allow the player to loot it in exchange for construction/maintenance tasks at their facility. Might be considered boring for many players, though.
I'd probably try to require the player to move a specific amount of stuff to a location by the facility (reachable by road), and the group would then move it to their base for use in the construction/maintenance they need done (plus some "profit"). That would mean the player would supply manpower for the tasks, but they would supply the tools and materials from their loot.

If you want to have combat missions you should probably turn things around, so the group would consist of preppers who have materials, equipment, and skills, but were the hiding type of preppers, not the suicidal fighting ones. In that scenario I can see them wanting your help to rescue members and do other kinds of fighting for them, and the rewards would be training (assuming training is made useful, not the current way of thirsting/heating to death spending a couple of days learning extremely little), or equipment, plus loot from the rescue location (minus at least some of the items the groups rescued were looking for).

**Comment by RanikOrtega** [comment]
@PatrikLundell Regarding point 1. My initial setup was that they were prepared and competent in combat, construction and survival but were severely undermanned since the cataclysm took 95% of their population. Hence making them open to outsiders that were not bandits. Discord convo had suggestions I alter it to "they need help with odd jobs" 

Regarding point 2. These were the last stand of the non-feral police forces of a large city. Everyone who still had their sanity retreated to somewhere near their HQ (Which was being overrun) that they felt was more defendable. So the story beats would go something like "Last we heard from them they were getting overrun at police HQ. They said they were pulling out and going to one of the more secure buildings near HQ. That leaves a few options. If you can make it to their last location then you should have some good gear to recover." The kicker is, one or two of the members may survive unknown to the player or NPC's. So when you go to just get loot from their last stand you can actually find a survivor. 


You may have a point of them offering combat missions if we lead from the hook of "We are capable but severely understrength compared to what we expected. Only 3 of our experienced members made it, the other few are young newbies who probably joined for the beer during our meetups. At this point we'd rather you risk your life than we risk ours. So we will let you in on some of our softer targets if you give us a cut." 

Actually, having it as a handful of experienced militia with a handful of either dependents or totally inexperienced newbies that didn't even bring that much gear is probably a good angle

**Comment by PGR-14** [comment]
Is there any work being done on this?

**Comment by RanikOrtega** [comment]
> Is there any work being done on this?

Yes here in [#73053](https://github.com/CleverRaven/Cataclysm-DDA/pull/73053) I've had a release ready build for over a week but I want to make the map design a bit more intricate.

**Comment by PGR-14** [comment]
Thank you.
